oracledb-ballerina/tests/resources/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2021, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Dockerfile using dockerhub repository image
+
+# ARG REPO_LOCATION=gayaldassanayake/database
+# ARG BASE_VERSION=19.3.0-ee
+ARG REPO_LOCATION=gayaldassanayake/oracle-enterprise
```

Comment:
Is it possible include what you have done to the base image in this docker file and use the original docker image?

Reply:
I could not find an Oracle docker instance that can be pulled using a dockerfile. All images they had needed manual accepting of a user terms before pulling.